packages/extension-ui/src/Popup/Derive/SelectParent.tsx

```diff
@@ -123,6 +141,30 @@ export default function SelectParent({
     passwordInputRef.current?.querySelector('input')?.focus();
   }, [_onParentPasswordEnter]);
 
+  const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
+
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  const _handleInputTypeChange = useCallback(() => {
+    setIsPasswordVisible(!isPasswordVisible);
+  }, [isPasswordVisible]);
+
+  const MIN_LENGTH = 6;
```

Comment:
I would put that a bit further up - good practice is to have const on the top. And MIN_LENGTH of what? Would be nice to make sure it's MIN_PASSWORD_LENGTH :)